src/sickle.ts

```diff
@@ -66,6 +77,34 @@ class Annotator {
           this.writeNode(node);
         }
         break;
+      case ts.SyntaxKind.InterfaceDeclaration:
+        this.writeRange(node.getFullStart(), node.getEnd());
+        if (node.flags & ts.NodeFlags.Ambient) {
+          // It's a "declare interface ..."; write a Closure interface
+          // declaration so we can @export all properties.
+          let decl = <ts.InterfaceDeclaration>node;
+          let className = decl.name.getText();
+          let props: TypedProperty[] = [];
+          for (let member of decl.members) {
+            // Members can include things like index signatures, for e.g.
+            //   interface Foo { [key: string]: number; }
+            // For now, just die unless all the members are regular old
+            // properties.
+            if (member.kind != ts.SyntaxKind.PropertySignature) {
```

Comment:
Possibly?  I think especially in the case where you're mixing index signatures and properties the mapping to Closure gets funky.